client/src/Dashboard/Dashboard.js

```diff
@@ -256,12 +258,19 @@ export function Dashboard() {
     }
   }
 
-  const getDashboardData = async (filterDate = undefined) => {
+  const getDashboardData = async ({
+    businessIds = [],
+    filterDate = undefined
+  } = {}) => {
     dispatch(setLoading(true))
-    const baseUrl = '/api/v1/case_list_for_dashboard'
+    let baseUrl = `/api/v1/case_list_for_dashboard`
```

Comment:
Why was this change needed? I don't see baseUrl changing anywhere. Maybe I'm missing something?

Reply:
This change wasn't needed, I had planed to change that more and then didn't need to.  It was an error on my part

Reply:
@fabianraf Is there any more changes you want me to make to get this PR approved?

Reply:
Sorry, I thought I had approved already. Approving it now!